plugins/src/main/kotlin/com/freeletics/gradle/setup/DaggerAnvilSetup.kt

```diff
@@ -36,8 +38,13 @@ internal fun Project.configureDagger(mode: DaggerMode) {
     }
 
     if (mode == DaggerMode.ANVIL_WITH_KHONSHU) {
-        val configuration = if (khonshuKsp.get()) {
-            configureProcessing(useKsp = true)
+        val configuration = if (anvilKsp.get()) {
+            configureProcessing(useKsp = true).also {
+                // TODO workaround for Gradle not being able to resolve this in the ksp config
+                configurations.named(it).configure {
+                    it.exclude(mapOf("group" to "org.jetbrains.skiko", "module" to "skiko"))
+                }
```

Comment:
This is the easiest fix for now. I'll look into not having the dependency at all in Khonshu separately